grudge/eager.py

```diff
@@ -251,6 +251,11 @@ def weak_div(self, *args):
 
     @memoize_method
     def normal(self, dd):
+        """Get unit normal to specified surface discretization, *dd*.
+
+        :arg dd: a :class:`~grudge.sym.DOFDesc` as the surface discretization.
+        :returns: surface normal as :class:`~pymbolic.geometric_algebra.MultiVector`.
```

Comment:
This doesn't seem right. While it's true that grudge relies on geometric algebra/exterior calculus internally, I'm not crazy enough to believe that I can get away with inflicting that on a broader user base. I just tried calling `discr.normal`, and I got an object array of `DOFArray`s back.